Makefile

```diff
@@ -3,6 +3,10 @@
 
 include ./static/tools/mk/lint.mk
 
+ifndef GOPATH
+$(error GOPATH not set. Please set GOPATH and make sure status-go is located at $$GOPATH/src/github.com/status-im/status-go.)
```

Comment:
Maybe we can also print a link to https://golang.org/doc/code.html#GOPATH?

Reply:
Sure. For reference, here is how the failure will look: 

```Bash
$ make
Makefile:7: *** GOPATH not set. Please set GOPATH and make sure status-go is located at $GOPATH/src/github.com/status-im/status-go. For more information about the GOPATH environment variable, see https://golang.org/doc/code.html#GOPATH.  Stop.
```